src/main/java/net/notfab/lindsey/core/commands/Invite.java

```diff
@@ -0,0 +1,59 @@
+package net.notfab.lindsey.core.commands;
+
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.entities.Member;
+import net.dv8tion.jda.api.entities.Message;
+import net.dv8tion.jda.api.entities.TextChannel;
+import net.notfab.lindsey.core.framework.command.Bundle;
+import net.notfab.lindsey.core.framework.command.Command;
+import net.notfab.lindsey.core.framework.command.CommandDescriptor;
+import net.notfab.lindsey.core.framework.command.Modules;
+import net.notfab.lindsey.core.framework.command.help.HelpArticle;
+import net.notfab.lindsey.core.framework.command.help.HelpPage;
+import net.notfab.lindsey.core.framework.i18n.Messenger;
+import net.notfab.lindsey.core.framework.i18n.Translator;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class Invite implements Command {
+
+    @Autowired
+    private Translator i18n;
+
+    @Autowired
+    private Messenger msg;
+
+    @Override
+    public CommandDescriptor getInfo() {
+        return new CommandDescriptor.Builder()
+            .name("invite")
+            .module(Modules.CORE)
+            .permission("commands.invite", "permissions.command")
+            .build();
+    }
+
+    @Override
+    public boolean execute(Member member, TextChannel channel, String[] args, Message message, Bundle bundle) throws Exception {
+        EmbedBuilder embed = new EmbedBuilder();
+        embed.setTitle(i18n.get(member, "commands.core.invite.title"));
+        embed.setThumbnail("https://gblobscdn.gitbook.com/spaces%2F-LSekC_Gg_7hjGlZBZoU%2Favatar.png");
```

Comment:
Instead of using a static url, change to `member#getGuild()#getSelfUser()#getEffectiveAvatarUrl()`.